test/e2e/run_transient_test.go

```diff
@@ -63,7 +63,7 @@ var _ = Describe("Podman run with volumes", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 
-		// All files should be in permament store, volatile
+		// All files should be in permanent store, volatile
```

Comment:
Uh, not actually part of this PR, but is there a missing "not" here ("**not** volatile")? I'm confused about the difference between this comment and the one on line 52.

Reply:
Me too, I just got these by running codespell, since I wanted to make sure I did not add a spelling mistake. I fixes the second one to be not volatile.